meerkat_frontend/src/js/technical/overview.js

```diff
@@ -18,6 +18,12 @@ function html_box_builder(overviewObj, locID) {
     // But default to a DIV table format.
     var html_base = overviewObj.html_base || "<div class='row' id ='" + overviewObj.parentId + "'></div>";
 
+	html_base = html_base.replace(/<th>([\s\S]*?)<\/th>/g, 
+								  function(match, label) {
```

Comment:
I've conceded that I was wrong about indentation when I started writing meerkat_frontend. I'm changing our javascript/html indentation to match our python indentation. I think you are still using tab characters perhaps?  Could you fix this as 4 spaces from now on?

Reply:
Interesting solution though - I was assuming they would have to be translated with the html embedded in the string - like the frontpage strings. If it works, nice!